src/features/weak-references.md

```diff
@@ -1,10 +1,13 @@
 ---
 title: 'Weak references and finalizers'
-author: 'Sathya Gunasekaran ([@_gsathya](https://twitter.com/_gsathya)) and Mathias Bynens ([@mathias](https://twitter.com/mathias))'
+author: 'Sathya Gunasekaran ([@_gsathya](https://twitter.com/_gsathya)), Mathias Bynens ([@mathias](https://twitter.com/mathias)), Shu-yu Guo ([@_shu](https://twitter.com/_shu)), and Leszek Swirski ([@leszekswirski](https://twitter.com/leszekswirski))
```

Comment:
@syg This is probably the reason - someone removed the closing quote :)

Reply:
That someone would be this doofus. :)